packages/launcher/src/renderer/launcher/contacts/ContactsView.js

```diff
@@ -158,11 +165,25 @@ export type Contact = {
   peerID: string,
   publicFeed: string,
   ethAddress?: string,
+  invite?: {
```

Comment:
These types should be imported from the files generated by Flow rather than created manually, it's easier to maintain when the queries are updated.